chrome/content/zotero/elements/contextPane.js

```diff
@@ -90,7 +90,7 @@
 				this._handleTabClose(action, type, ids, extraData);
 				return;
 			}
-			if (type == 'tab' && action == 'select') {
+			if (type == 'tab' && ["select", "load"].includes(action)) {
```

Comment:
This doesn't seem right. Won't this cause `_handleTabSelect()` to be called twice in rapid succession when you click on a tab?

Reply:
Might be from using `Zotero_Tabs.selectedID`. Updated to use the `extraData[ids[0]].type` from the notifier callback arguments, which will be called only once for each tab.